packages/utils/ast-utils.ts

```diff
@@ -605,7 +605,7 @@ function parseTopScope(j: JSCodeshift, ast: Node, value: string[], action: strin
  */
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-function parseMerge(j: JSCodeshift, ast: Node, value: string, action: string): boolean | Node {
+function parseMerge(j: JSCodeshift, ast: Node, value: string[], action: string): boolean | Node {
 	function createMergeProperty(p: Node): boolean {
 		// FIXME Use j.callExp()
```

Comment:
Split into two variables, give them meaningful names. First one is supposed to go here, and then you'll need to check if the require of this is present in the outer body of the AST. If it's present, you don't need to do anything, if it's not existent you will need to add it. 

Remove the topScope push and manually add that here. You could reuse the logic used in topScope to a certain extent. Remember to abstract this code so it looks neat